tests/test_util.py

```diff
@@ -23,3 +24,11 @@ def test_get_time_without_tz():
 
     assert 'not in UTC' in str(excinfo.value)
 
+
+def test_find_closest_point():
+    actual_distances, actual_index = (
+        [[118.77417243, 111.22983323],[122.9552163 , 115.67902656]],
+        1)
+    expected_distances, expected_index = find_closest_point([-99.83, -99.32], [42.25, 42.21], -99.32, 43.25)
```

Comment:
I think actual and expected are normally swapped, as in: expected is what you expect the function will return. Actual (or: `result`) is what the function actually returned